soh/soh/z_message_OTR.cpp

```diff
@@ -187,4 +187,142 @@ extern "C" void OTRMessage_Init()
         CustomMessage("Hey! Hey!&You can't take the rod out of here!&I'm serious!^Do you want to quit?&\x1B&%gYes&No%w",
                       "Hey! Hey!&Du kannst die Angel doch nicht&einfach mitnehmen!&Ganz im Ernst!^Möchtest du aufhören?&\x1B&%gJa&Nein%w", //TODO Used AI translation as placeholder
                       "Holà! Holà!&Les cannes ne sortent pas d'ici!&Je suis sérieux!^Voulez-vous arrêter?&\x1B&%gOui&Non%w")); //TODO Used AI translation as placeholder
-}
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_GREETING,
+        CustomMessage("Hey there, @! You&making a deposit? Or are you&here to make a withdrawal?",
+                    "Hallo, @! Machen Sie&eine Einzahlung? Oder sind Sie&hier, um eine Auszahlung&vorzunehmen?",
+                    "Salut, @! Vous&effectuez un dépôt ? Ou êtes-vous&ici pour effectuer un retrait?"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_OPTIONS,
+        CustomMessage("What would you like to do?" "\x1C" "&%gDeposit Rupees&%gWithdrawal Rupees&%gNothing%w",
+                    "Was würdest du gern tun?" "\x1C" "&%gRubine einzahlen&%gRubine abheben&%gNichts%w",
```

Comment:
Considering the majoritarian usage of the polite form in the added dialogue lines, I would suggest to standardize the German dialogue in a consistent way.

`Was würdest du gern tun?` &#8594; `Was würden Sie gern tun?`